FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Hardeep Singh on 15/09/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImagesMapper {
+	private struct Root: Codable {
+		let items: [Item]
+		var images: [FeedImage] {
+			items.compactMap { $0.image }
+		}
+	}
+
+	private struct Item: Codable {
+		let image_id: UUID
+		let image_url: String
+		let image_loc: String?
+		let image_desc: String?
+
+		var image: FeedImage? {
+			guard let imageURL = URL(string: image_url) else {
+				return nil
+			}
+			return FeedImage(id: image_id,
+			                 description: image_desc,
+			                 location: image_loc,
+			                 url: imageURL)
+		}
+	}
```

Comment:
`URL` [conforms](https://github.com/apple/swift/blob/d1f798bedb61d4c137573360e6e623670efedc2a/stdlib/public/Darwin/Foundation/URL.swift#L1201-L1227) to `Codable`, there's no need to create it manually from string.
```suggestion
	private struct Item: Codable {
		let image_id: UUID
		let image_url: URL
		let image_loc: String?
		let image_desc: String?

		var image: FeedImage? {
			return FeedImage(id: image_id,
			                 description: image_desc,
			                 location: image_loc,
			                 url: image_url)
		}
	}
```